tsl/test/sql/continuous_aggs.sql

```diff
@@ -1393,14 +1393,34 @@ GROUP BY time_bucket('1week', timec), location;
 
 SELECT * FROM mat_m1 ORDER BY 1, 2;
 
+-- ORDER BY in the view definition
+DROP MATERIALIZED VIEW mat_m1;
```

Comment:
also need a test that shows enabling/disabling realtime aggregates with order by works correctly.

Reply:
Added!